index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, intial-scale=1.0"/>
+    <title>First Portfolio</title>
+    <link href="./style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!-- START NAVBAR SECTION-->
+    <header>
+        <nav>
+            <div class="navbar">
+                <button type="button" class="nav-btn">
+                    <div class="lines line1"></div>
+                    <div class="lines line2"></div>
+                    <div class="lines line3"></div>
+                </button>
+            </div>
+        </nav>
+    </header>
+    <!-- END NAVBAR SECTION-->
+
+    <!-- START HEADLINE SECTION-->
+    <section class="headline">
+        <div class="content">
+            <h1 class="author">Charlie Giorgania</h1>
```

Comment:
- [ OPTIONAL] As this is your portfolio, I suggest using your name instead of using the name given by the designer, kindly use your name :wink:  remove Charlie Giorgania and put your name. :smile: